### PR TITLE
Update price from CoinGecko

### PR DESCRIPTION
**Pull request by boyuan-chen**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Describe what your Pull Request is about in a few sentences.

The on-chain price oracle is out of date. This PR uses the price oracle from CoinGecko.

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

- Load price from CoinGecko

## Testing

Describe how to test your new feature/bug fix and if possible, a step by step guide on how to demo this.

### CODE REVIEW

File: packages/boba/gas-price-oracle/src/service.ts [hunk 1]
```diff
@@ -468,12 +498,30 @@ export class GasPriceOracleService extends BaseService<GasPriceOracleOptions> {
         network: 'L2',
         data: {
           /* eslint-disable */
```
Comment (jyellick) [inline packages/boba/gas-price-oracle/src/service.ts]: Do these formatting changes fix the eslint? Could we remove this disable? (Same with the others)

**Review by jyellick — APPROVED** [review]
Overall, LGTM -- but

File: packages/boba/gas-price-oracle/src/service.ts [hunk 2]
```diff
@@ -710,13 +759,28 @@ export class GasPriceOracleService extends BaseService<GasPriceOracleOptions> {
...
+  // Data provided by CoinGecko
+  private async _getTokenPriceFromCoinGecko(id: string): Promise<number> {
+    try {
+      const URL = `https:///api.coingecko.com/api/v3/coins/${id}?localization=false&tickers=false&community_date=false&developer_data=false&sparkline=false`
+      const payload = await fetch(URL)
+      const payloadParsed = await payload.json()
+      return Number(payloadParsed.market_data.current_price.usd)
+    } catch (err) {
+      this.logger.warn(`CAN\'T QUERY TOKEN PRICE ${err} - ${id} FROM CoinGecko`)
+      return 0
```
Comment (mmontour1306) [inline packages/boba/gas-price-oracle/src/service.ts:772]: Is it safe to return a value of 0 here? Does the rest of the code interpret it as an error condition?

Comment (boyuan-chen) [reply packages/boba/gas-price-oracle/src/service.ts:772]: Fixed.

File: packages/boba/gas-price-oracle/src/service.ts (hunk 1 above)
Comment (boyuan-chen) [reply packages/boba/gas-price-oracle/src/service.ts]: Removed

**Review by mmontour1306 — APPROVED** [review]

**Review by souradeep-das — COMMENTED** [review]
lgtm

**Pull request merged**